File: Sum_Of_Squares/Sum_Of_Squares.kt
import java.util.*

fun main(args: Array<String>) {
    val read = Scanner(System.`in`)
    print("Enter the Number : ")
    var num = read.nextInt()
    var ans = ((num) * (num + 1) * (2 * num + 1))/6
    println("The sum of square from 1 to N number is : ${ans}")
}

/*
    Input : Enter the Number : 223
    Output : The sum of square from 1 to N number is : 140
*/

File: Sum_Of_Squares/Sum_Of_Squares.py
'''
    The sum of squares of first N natural numbers is given
    by the equation ((N) * (N + 1) * (2 * N + 1)) / 6 i.e.
    1^2 + 2^2 + 3^2 + 4^2 + 5^2.........N^2 sums upto
    ((N) * (N + 1) * (2 * N + 1)) / 6
'''

print("Enter the value of N: ")
N = int(input())
ans = ((N) * (N + 1) * (2 * N + 1)) / 6
print(ans)

'''
Input:
N = 5

Output:
ans = 55

Verification:
1^2 + 2^2 + 3^2 + 4^2 + 5^2
= 1 + 4 + 9 + 16 + 25
= 55
'''
